### PR TITLE
hold/double actions for light-card

### LINKED ISSUE #5319: On-hold action on the light card.
## The request
  I would like to have a hold action on the light card.
  I have some lights created by light platform group that contain multiple light entities.
  My office has a spot with 3 bulbs. Bulbs are named light.office_bulbX (1,2,3) and with a group
  i create a entity called light.office to switch all 3 at once.
  i use a custom:slider-entity-row to show light.office.
  the toggle is used to turn on and off, a single click opens the default history popup
  and a on_hold action it shows a browser_mod popup that shows a custom:slider-entity-row
  for the 3 lights that are in there, so i can manage them individually.
  this works great. But i wanna start using the "new" Light card.
  But this has no on_hold option, so no way to get a popup with the single bulbs.

## The alternatives
  No alternatives

### PR DESCRIPTION
**Pull request by iantrich**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5319
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts [hunk 1]
```diff
@@ -56,11 +63,28 @@ export class HuiLightCardEditor extends LitElement
...
     }
 
+    const actions = [
+      "more-info",
+      "toggle",
+      "navigate",
+      "url",
+      "call-service",
+      "none",
+    ];
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: Why haven't we extracted this somewhere? The actions are mainly the same for all cards?

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts [hunk 2]
```diff
@@ -108,6 +132,32 @@ export class HuiLightCardEditor extends LitElement
...
+        ></hui-action-editor>
+
+        <hui-action-editor
+          .label="${this.hass.localize(
+            "ui.panel.lovelace.editor.card.generic.double_tap_action"
+          )} (${this.hass.localize(
+            "ui.panel.lovelace.editor.card.config.optional"
+          )})"
+          .hass=${this.hass}
+          .config="${this._double_tap_action}"
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: Let's make it value for all editors so we don't have to do multiple checks in the event
```suggestion
          .value="${this._double_tap_action}"
```

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts [hunk 3]
```diff
@@ -108,6 +132,32 @@ export class HuiLightCardEditor extends LitElement
...
+          .actions="${actions}"
+          .configValue="${"double_tap_action"}"
+          @action-changed="${this._valueChanged}"
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: And use standard events
```suggestion
          @value-changed="${this._valueChanged}"
```

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts (hunk 1 above)
Comment (iantrich) [reply src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: I can do that in another PR

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts (hunk 3 above)
Comment (iantrich) [reply src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: I can do that in another PR

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts (hunk 2 above)
Comment (iantrich) [reply src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: I'll do that in another PR

**Pull request merged**